packages/sources/css/src/components/rating/src/index.css

```diff
@@ -0,0 +1,149 @@
+}
+
+/***** Highlight Stars on Hover *****/
+
+.vtmn-rating--interactive > input:checked ~ .full::before, /* show gold star when clicked on full */
+.vtmn-rating--interactive:not(:checked) > .full:hover::before, /* hover current full star */
+.vtmn-rating--interactive:not(:checked) > label:hover ~ label::before {
+  /* hover previous stars in list */
+  content: url("data:image/svg+xml;charset=utf-8, <svg width='16' height='16' viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'><path fill-rule='evenodd' clip-rule='evenodd' d='m8 12.173-4.702 2.632 1.05-5.285L.391 5.861l5.352-.634L8 .333l2.257 4.894 5.352.634-3.957 3.659 1.05 5.285L8 12.173z' fill='%23007dbc'/></svg>");
+}
+
+.vtmn-rating_note,
+.vtmn-rating_comment {
+  font-family: var(--vtmn-typo_font-family);
+  font-size: var(--vtmn-typo_text-2-font-size);
+  color: var(--vtmn-color_black);
+  font-weight: var(--vtmn-typo_font-weight--bold);
+}
+
+.vtmn-rating_comment--secondary {
+  font-family: var(--vtmn-typo_font-family);
+  font-size: var(--vtmn-typo_text-2-font-size);
+  font-weight: var(--vtmn-typo_font-weight--regular);
+  color: var(--vtmn-color_grey);
+}
+
+.vtmn-rating span[class^='vtmx-'] {
+  font-size: var(--vtmn-typo_text-2-font-size);
+}
+
+/* Color variants */
+
+.vtmn-rating_color--brand {
+  color: var(--vtmn-color_brand-digital);
+}
+
+.vtmn-rating_color--black {
+  color: var(--vtmn-color_black);
+}
+
+/* Size variants */
+
+.vtmn-rating_size--small {
+  font-size: var(--vtmn-typo_text-3-font-size);
+}
+
+.vtmn-rating_size--small span {
+  font-size: var(--vtmn-typo_text-3-font-size);
+}
+
+.vtmn-rating_size--small span[class^='vtmx-'] {
+  font-size: var(--vtmn-typo_text-2-font-size);
+}
+
+.vtmn-rating_size--medium {
+  font-size: var(--vtmn-typo_text-1-font-size);
+}
+
+.vtmn-rating_size--medium span {
+  font-size: var(--vtmn-typo_text-1-font-size);
+}
+
+.vtmn-rating_size--medium span[class^='vtmx-'] {
+  font-size: rem(32px);
+}
+
+.vtmn-rating.vtmn-rating_size--small > * {
+  margin: 0 rem(3px);
+}
+
+.vtmn-rating_size--medium .vtmn-rating--interactive > label::before {
+  margin: 0 rem(5px);
+}
+
+.vtmn-rating.vtmn-rating_size--medium > * {
+  margin: 0 rem(5px);
+}
+
+.vtmn-rating_size--medium .vtmn-rating--interactive > label {
+  transform: translateY(rem(2px));
+}
+
+.vtmn-rating_size--small .vtmn-rating--interactive > label::before {
+  content: url("data:image/svg+xml;charset=utf-8, <svg width='16' height='16' viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'><path fill-rule='evenodd' clip-rule='evenodd' d='M3.298 14.805 8 12.173l4.702 2.632-1.05-5.285 3.957-3.659-5.352-.634L8 .333 5.743 5.227.39 5.86l3.958 3.66-1.05 5.285zm7.533-2.575L8 10.645 5.169 12.23 5.8 9.048 3.42 6.845l3.222-.382L8 3.517l1.359 2.946 3.222.382-2.382 2.203.632 3.182z' fill='%23007dbc'/></svg>");
```

Comment:
We will not be able to merge this if it's impossible to change color dynamically with CSS Custom Properties :(

Reply:
Ok, I'll have a look at postCSS plugins to prevent this use of SVG

Reply:
Done ! Work great with masks